### ISSUE TITLE
Position of buttons

### ISSUE THREAD

**Issue by HimangshuRanjan** [EternoSeeker/gameoflife#225]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Problem👇
![Screenshot 2024-06-07 235104](https://github.com/EternoSeeker/gameoflife/assets/146883557/22d09832-09e2-4267-a30f-8fbdc2259178)

Solution👇
![Screenshot 2024-06-08 000028](https://github.com/EternoSeeker/gameoflife/assets/146883557/9a2ca061-7ea7-4524-9ea2-3d914a8f8ef2)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by EternoSeeker** [comment]
yes, also if you could reduce length of the grid size input box and the length of change grid size button, it would look better
